addons/web/static/src/xml/base.xml

```diff
@@ -461,12 +461,13 @@
 <t t-name="web.OwlDialog" owl="1">
     <Portal target="'body'">
         <div class="o_dialog" t-on-focus="_onFocus" t-on-click="_onClick">
-            <div t-if="props.backdrop" class="modal-backdrop show" t-on-click="_onBackdropClick"/>
             <div role="dialog" class="modal"
                 tabindex="-1"
                 t-att-class="{ o_technical_modal: props.technical, o_modal_full: props.fullscreen }"
                 t-ref="modal"
+                t-att-data-backdrop="props.backdrop"
```

Comment:
I don't know why I hadn't already done it and why it worked before without it but I added this line here